src/MICore/osxlaunchhelper.scpt

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/osascript
+
+on run argv
+   set debuggerTitle to (item 1 of argv)
+   set executeCommand to (item 2 of argv)
+   # Note: if other tabs are open on the terminal window that is opened by this script, this won't behave properly.
+   set command to "clear; " & ¬
```

Comment:
There is kind of a weird character here. Just wanted to make sure -- this is intentional correct?